### PR TITLE
Display the scopes field in m2m guide only when API's are authorized

### PR DESCRIPTION
**Pull request by asha15**
### Purpose
$subject

### Related Issues
- 

### Related PRs
- 

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: features/admin.extensions.v1/application-templates/shared/components/m2m-custom-configuration.tsx [hunk 1]
```diff
@@ -168,6 +168,10 @@ export const M2MCustomConfiguration: FC<M2MCustomConfigurationPropsInterface> =
         tokenRequest = tokenRequest.concat(` -d 'scope=${copyScopesValue}'`);
     }
 
+    const isScopeAvailable = (value: string) => {
```
Comment (brionmario) [inline features/admin.extensions.v1/application-templates/shared/components/m2m-custom-configuration.tsx:171]:
```suggestion
    const isScopeAvailable = (value: string): string => {
```

**Review by brionmario — APPROVED** [review]

File: features/admin.extensions.v1/application-templates/shared/components/m2m-custom-configuration.tsx [hunk 2]
```diff
@@ -250,36 +254,38 @@ export const M2MCustomConfiguration: FC<M2MCustomConfigurationPropsInterface> =
...
-                                                    "extensions:develop.applications.quickstart" +
-                                                    ".m2m.configurations.scopes"
-                                                }
-                                            >
-                                                The list of authorized scopes.
-                                                Include required scopes by authorizing APIs through the
-                                                <a
-                                                    className="link pointing"
-                                                    onClick={ onAPIAuthorizationTabClick }
+                                { isScopeAvailable(copyScopesValue) && (
```
Comment (pavinduLakshan) [inline features/admin.extensions.v1/application-templates/shared/components/m2m-custom-configuration.tsx:257]: can't we simply use `!!copyScopesValue` here?

```suggestion
                                { !!copyScopesValue && (
```

Comment (pavinduLakshan) [reply features/admin.extensions.v1/application-templates/shared/components/m2m-custom-configuration.tsx:257]: Also, have we considered showing a warning/info message when scopes field is not available?

Comment (asha15) [reply features/admin.extensions.v1/application-templates/shared/components/m2m-custom-configuration.tsx]: Fixed with: https://github.com/wso2/identity-apps/pull/6471/commits/2dd27a320bfa6287ea8a9fda305d7b85649a74f2

Comment (asha15) [reply features/admin.extensions.v1/application-templates/shared/components/m2m-custom-configuration.tsx]: `have we considered showing a warning/info message when scopes field is not available?` Previously we were displaying an info message with the empty field, but then it was decided to remove the scope field when API's are not authorized.

**Review by pavinduLakshan — APPROVED** [review]

**Pull request merged**